### ISSUE TITLE
Stops tracking after a while

### ISSUE THREAD

**Issue by alanhamlett** [wakatime/browser-wakatime#192]
Not sure why, but after some time the extension stops tracking until the browser is restarted.

**Comment by jvelezpo** [comment]
I havent been able to reproduce this bug on my end.
Sent these steps to Carlos Ortiz 

```
Can you please test following these steps
Open the extension options,
debug add on
then inspect wakatime
After this, take a look in the network tab, when changing tabs in the browser you should see a heartbeat post request to wakatime API
```


https://user-images.githubusercontent.com/1050904/234432656-24e03333-b0db-4bb0-8fbf-e3acf48132c2.mp4

If the POST request are not there, then we will be able to see any error in the console

**Comment by alanhamlett** [comment]
Should be fixed by #208.
